crates/api/src/site/purge/person.rs

```diff
@@ -31,8 +31,14 @@ pub async fn purge_person(
 
   // Read the local user to get their images, and delete them
   if let Ok(local_user) = LocalUserView::read_person(&mut context.pool(), data.person_id).await {
-    let pictrs_uploads =
-      LocalImage::get_all_by_local_user_id(&mut context.pool(), &local_user.local_user.id).await?;
+    let pictrs_uploads = LocalImage::get_all_by_local_user_id(
+      &mut context.pool(),
+      local_user.local_user.id,
+      None,
+      None,
+      true,
+    )
+    .await?;
 
     for upload in pictrs_uploads {
       delete_image_from_pictrs(&upload.pictrs_alias, &upload.pictrs_delete_token, &context)
```

Comment:
Not part of this PR, but does this make a DB call for each image? If so, it seems computationally expensive, especially if purging an account that has been around for awhile and posted a lot of images.

Reply:
This one is just sending http deletes to the locally runinng pictrs, so it has to loop.